tiledb/sm/cpp_api/enumeration_experimental.h

```diff
@@ -223,6 +223,24 @@ class Enumeration {
   /*          STATIC FUNCTIONS         */
   /* ********************************* */
 
+  /**
+   * Create an empty enumeration.
+   *
+   * @param ctx The context to use.
+   * @param ordered Whether or not to consider this enumeration ordered.
```

Comment:
Missing params. Could you fix the function below as well for missing @name?

Reply:
Fixed up all the inconsistencies I found. Let me know if I've missed any.